ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -264,3 +267,15 @@ def map_service(geocat_service, issued, modified, description, rights):
         'media_type': geocat_service.get('media_type', ''),
         'url': geocat_service.get('url', '')
     }
+
+
+def _map_geocat_resource_format_to_valid_format(geocat_format):
+    valid_formats = mu.get_format_values()
+    for key, value in valid_formats.items():
+        if geocat_format == key:
+            return value
+    valid_media_types = mu.get_iana_media_type_values()
+    for key, value in valid_media_types.items():
+        if geocat_format == key:
+            return value
+    return geocat_format
```

Comment:
As we do not have the mapping rules for the geocat values yet, I added as default return the geocat value for the mapping method, so that we can develop further without breaking everything and will need less time for adding the mapping, until it is approved by geocat, WDYT?